demisto_sdk/__main__.py

```diff
@@ -3198,9 +3211,136 @@ def update_content_graph(
+    help="The path to the input file to run the command on.",
+    multiple=True,
+    type=click.Path(path_type=Path),
+)
+@click.option(
+    "-g",
+    "--git-diff",
+    help="Whether to use git to determine which files to run on",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-a",
+    "--all-files",
+    help="Whether to run on all files",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-ut",
+    "--unit-test",
+    help="Whether to run unit tests for content items",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--skip",
+    help="A comma separated list of precommit hooks to skip",
+)
+@click.option(
+    "--validate",
+    help="Whether to run demisto-sdk validate",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--format",
+    help="Whether to run demisto-sdk format",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--native-images",
+    help="Whether to run demisto-sdk test on native images",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "-v",
+    "--verbose",
+    help="Verbose output of pre-commit",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--show-diff-on-failure",
+    help="Show diff on failure",
+    is_flag=True,
+    default=False,
+)
+@click.option(
+    "--sdk-ref",
+    help="The demisto-sdk ref to use for the pre-commit hooks",
+    default="",
+)
+@click.pass_context
+@logging_setup_decorator
+def pre_commit(
+    ctx,
+    input: Iterable[Path],
+    git_diff: bool,
+    all_files: bool,
+    unit_test: bool,
+    skip: str,
+    validate: bool,
+    format: bool,
+    native_images: bool,
+    verbose: bool,
+    show_diff_on_failure: bool,
+    sdk_ref: str,
+    **kwargs,
+):
+    from demisto_sdk.commands.pre_commit.pre_commit_command import pre_commit_manager
+
+    if skip:
+        skip = skip.split(",")  # type: ignore[assignment]
+    sys.exit(
+        pre_commit_manager(
+            input,
+            git_diff,
+            all_files,
+            unit_test,
+            skip,
+            validate,
+            format,
+            native_images,
+            verbose,
+            show_diff_on_failure,
+            sdk_ref=sdk_ref,
+        )
+    )
+
+
+@main.command(short_help="Run unit tests in a docker for integrations and scripts")
+@click.help_option("-h", "--help")
+@click.option(
+    "-i",
+    "--input",
+    type=PathsParamType(
+        exists=True, resolve_path=True
+    ),  # PathsParamType allows passing a list of paths
+    help="The path of the content pack/file to validate specifically.",
+)
+@click.option(
+    "-v", "--verbose", is_flag=True, default=False, help="Verbose output of unit tests"
+)
+@click.argument("file_paths", nargs=-1, type=click.Path(exists=True, resolve_path=True))
+@click.pass_context
+@logging_setup_decorator
+def run_unit_tests(input: str, file_paths: Tuple[str, ...], verbose: bool):
+    if input:
+        file_paths = tuple(input.split(","))
```

Comment:
what if both `input` and `file paths` are used?

Reply:
input wins

Reply:
I suggest raising an error to avoid confusion